data/human/jobs.txt

```diff
@@ -3781,3 +3781,814 @@ mission "Hunted"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on complete
+		payment 15000 180
+		dialog phrase "generic passenger dropoff payment"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+mission "Syndicate Defectors [1]"
+	name "Syndicate defectors to <planet>"
+	job
+	repeat
+	description "Go to <stopovers> and pick up <bunks> Syndicate defectors. Fly them to <destination> for <payment>. Expect bounty hunters."
+	passengers 4 15 .8
+	clearance "You provide a passcode given to you on <origin> and the authorities give you landing coordinates."
+	to offer
+		random < 50
+		"passenger space" > 5
+	source
+		government "Humanika (Port)"
+	destination
+		government "Humanika (Port)"
+	stopover
+		government "Pirate"
+		not attributes uninhabited
+		distance 5 16
+	on visit
+		dialog phrase "generic passenger on visit"
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on complete
+		payment 15000 180
+		dialog phrase "generic passenger dropoff payment"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+mission "Syndicate Defectors [2]"
+	name "Syndicate defectors to <planet>"
+	job
+	repeat
+	description "Go to <stopovers> and pick up <bunks> Syndicate defectors. Fly them to <destination> for <payment>. Expect bounty hunters."
+	passengers 7 28 .8
+	clearance "You provide a passcode given to you on <origin> and the authorities give you landing coordinates."
+	to offer
+		random < 40
+		"passenger space" > 10
+	source
+		government "Humanika (Port)"
+	destination
+		government "Humanika (Port)"
+	stopover
+		government "Pirate"
+		not attributes uninhabited
+		distance 5 16
+	on stopover
+		dialog phrase "generic passenger stopover pickup"
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 15000 180
+		dialog phrase "generic passenger dropoff payment"
+	npc
+		government "Bounty Hunter"
+		personality heroic unconstrained nemesis plunders harvests waiting
+		fleet "Bounty Hunters"
+		system
+			distance 1 2
+
+# Can't do phrases in the right places for this yet:
+phrase "syndicate defectors"
+	word
+		"religious" 3
+		"overworked" 2
+		"persecuted" 3
+		"abused" 2
+	word
+		" Syndicate "
+	word
+		"union organizers" 3
+		"low-level managers" 2
+		"workers" 5
+		"human rights observers"
+		"managers"
+		"engineers"
+		"security personnel"
+		"technicians"
+
+# NOTE: Slavery rescue missions can have an uninhabited Pirate planet
+# stopover since that is a valid place for slaves to have escaped to.
+
+mission "Rescue Slaves [0]"
+	name "Help slaves escape to <planet>"
+	job
+	repeat
+	description "Slaves with valuable skills are asking for rescue. Pick up all <bunks> of them on <stopovers> and take them to them <destination> for <payment>. Expect bounty hunters."
```

Comment:
> "<payment>" - Who's paying us?

New Tortuga arranged for the transportation, but they want to remain anonymous to avoid hurting their relationship with local pirate groups. Openly challenging pirates will draw a stronger response than sneaky stealing, since pirates sneakily steal from each other all the time.

> "... are asking for rescue." - I imagine a lot of them are! Why are we getting a job board message for this group in particular?

These arrangements are extraordinarily expensive (look at one in the job board and you'll see). Rescuing millions of slaves this way is not sustainable. So, they rescue "high-value" slaves.

> Where's the rest of the underground railroad? 

That is not covered by this PR.

However, in Hai Reveal, you do learn of one "underground railroad." It is run by the Syndicate, to rescue slaves and give them Syndicate jobs. The person who tells you genuinely believes this is benevolent, so it isn't intended as a transfer from one slavery to another. Obviously, New Tortuga would not be involved in such a scheme, since many people on that planet would view it as another form of slavery.

> "Expect bounty hunters." - On-planet, where I'm surprisingly vulnerable, or in space, where I fly around in super powerful stolen alien warships?

Yes.

> What allies do we have in this mission? Am I going to have to lay siege to a pirate fortress myself, or has someone already freed the people from their imprisonment?

I'll update this to say you're picking them up at a predetermined location.

Reply:
How is this?

![escaped-slaves-try3](https://github.com/endless-sky/endless-sky/assets/116329264/6d26b903-355c-4418-a864-3eee848677f5)

Reply:
For all my criticisms, I do genuinely enjoy this part of the PR. About damn time First Last did something about the interstellar slave trade.

I dig it.